.github/ISSUE_TEMPLATE/Questions.md

```diff
@@ -4,4 +4,13 @@ about: Ask questions about the library
 
 ---
 
+Learn more about how your form project could leverage the US Forms System. To help all of us use our time most efficiently:
 
+- **What is the scope of your project?**

```

Comment:
Can we define "scope" here? I'm not sure how I would answer this question myself.